dev/com.ibm.ws.jbatch.security/src/com/ibm/ws/jbatch/security/impl/WSBatchAuthServiceImpl.java

```diff
@@ -448,5 +572,58 @@ private String getFormattedMessage( String msgId, Object[] fillIns, String defau
                 defaultMsg);
     }
 
+	@Override
+	public Set<String> getGroupsForSubject() {		
+
+		return getSubjectGroups(runAsSubject());
+		
+	}
+	
+	@Override
+	public Set<String> normalizeGroupNames(Set<String> groupNames) {
```

Comment:
I might have put this method over in the CL side. It's only used when persisting, so we could have just done the normalize first and then call "update" with the normalized values.    

I realize it's non-trivial to move it.. you might have to redo bundle imports.   At this point it's even harder since you might have a multi-repo delivery to stage.   So... we probably won't move it, but if for some reason it becomes easier maybe do that.